### PR TITLE
Added FAQ Section

### PR DESCRIPTION
**Pull request by AlephNul1**
Added FAQ

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+### What really is Cryptocurrency
+- Maybe we should start with what it is not... It is not an esoteric one percenters currency, it is not a underground criminal money system, it is not using more power than the current fiat system, it is not hard to use, and above all... it does not require selling your soul and identity to the state to use and gain access to.
+- For a comprehensive look into what it is check out [3Blue1Brown - But how does bitcoin actually work?](https://www.youtube.com/watch?v=bBC-nXj3Ng4) This video goes rather in depth and is a very good explanation over all.
+### What is Monero
```
Comment (onionltd) [inline content/faq.md:4]: All questions should end with a question mark.

File: content/faq.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- First and foremost Monero is a private, decentralized cryptocurrency that keeps your finances confidential and secure. Head to [GetMonero](https://www.getmonero.org/) for more information
+- Get Monero also has a nice [Moneropedia](https://www.getmonero.org/resources/moneropedia/) for more help
+### What is KYC-Free 
+- KYC stands for Know Your Client (or Customer) and is a basically a federal or state level mandate that specific institutions are required by law to put "effort" into validating the identity of its customers. This is normally specific to ID verification. The term KYC-Free will be used quite often in conversations around cryptocurrency and Monero specifically so it is good to know. 
+- KYC-Free is just a way to let people know that a specific services does not require ID verification. This does not mean it may not ask for an email or phone number but those can be created and thrown away easily unlike a state issued ID.
```
Comment (onionltd) [inline content/faq.md:9]: I don't see how this is relevant to Monero gift cards.

File: content/faq.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+### How do I use my Gift card
+- <b>WARNING: The Gift card has the information to restore and use a wallet. This information is supposed to be known only by you otherwise someone else can use your funds. The moment you restore your wallet AKA Redeem your gift card, you should immediately move the funds to your own personal wallet. You must assume that since the Gifter knows the Seed (Since they printed it) that you now have no control on knowing who else could know it.</b>
```
Comment (onionltd) [inline content/faq.md:11]: No need for HTML tags. Use `**` for bold.

File: content/faq.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+- This project is still rather new, so depending on who gave you the card what the QR code says could be different. There are a few possible situations
+- Mnemonic Seed and Block Height
+	- If your QR code gives you a 25 word Seed and a Block Height Number you should be able to just copy and paste this information into the "Restore Wallet" option 
+    - If your QR code gives a URI scheme then you might be limited at this time on what can use it quickly, we are requesting several wallets to support restoring using this format.
+### What wallets should I use
+- The Gift Card should have some options for you as well as more at [GetMonero](https://www.getmonero.org/)
+- Always validate your downloads, a tampered wallet download could result in loss of funds
+### Where can I use Monero
+- Moreno is not as widely accepted as Bitcoin is (which we hope to change) but you may find some out there that accept it. Apart from using it to trade between friends or on Peer to Peer markets online, you can also swap it for Bitcoin using some of the below options and then send that.
```
Comment (onionltd) [inline content/faq.md:20]: Typ Moreno -> Monero.

File: content/faq.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+### What really is Cryptocurrency
```
Comment (onionltd) [inline content/faq.md:1]: I would not include this question on FAQ. It's impossible to give a short comprehensive answer to what is a cryptocurrency. The answer to "What is Monero" is sufficient, in my opinion as it outlines the most important features of Monero a new user should know. The rest is just technicalities.

File: content/faq.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+### How do I use my Gift card
+- <b>WARNING: The Gift card has the information to restore and use a wallet. This information is supposed to be known only by you otherwise someone else can use your funds. The moment you restore your wallet AKA Redeem your gift card, you should immediately move the funds to your own personal wallet. You must assume that since the Gifter knows the Seed (Since they printed it) that you now have no control on knowing who else could know it.</b>
+- This project is still rather new, so depending on who gave you the card what the QR code says could be different. There are a few possible situations
+- Mnemonic Seed and Block Height
+	- If your QR code gives you a 25 word Seed and a Block Height Number you should be able to just copy and paste this information into the "Restore Wallet" option 
+    - If your QR code gives a URI scheme then you might be limited at this time on what can use it quickly, we are requesting several wallets to support restoring using this format.
+### What wallets should I use
```
Comment (onionltd) [inline content/faq.md:16]: Can we just link to getmonero.org/downloads/#mobilelight here?

File: content/faq.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+- The Gift Card should have some options for you as well as more at [GetMonero](https://www.getmonero.org/)
+- Always validate your downloads, a tampered wallet download could result in loss of funds
```
Comment (onionltd) [inline content/faq.md:18]: > Always validate your download, [...]

This is confusing. Most people are going to use a mobile wallet. There's no need/way to verify the downloaded app.

**Comment by AlephNul1** [comment]
I am going to take this back into the shop and work on it some more based on your notes.
Thanks

**Pull request closed without merging**